recipe/meta.yaml

```diff
@@ -10,27 +10,39 @@ source:
   sha256: 42f3dbe39bd1618a98f1b9ea43ac2573533fc56badb85fee5145d1235467f636
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  number: 1
 
-requirements:
-  host:
-    - pip
-    - python >=3.9
-  run:
-    - python >=3.9
-    - bokeh ==3.*
-    - ipykernel ==6.*,!=6.18.0
-    - ipywidgets ==8.*
+outputs:
+  - name: ipywidgets_bokeh
+    script: {{ PYTHON }} -m pip install . -vv
```

Comment:
I get a `PYTHON is undefined` error [here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=935804&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3&l=508), and I have no idea why.

@jakirkham, is it possible for you to help me with this?

Reply:
I found a way by taking inspiration from https://github.com/conda-forge/typing_extensions-feedstock/blob/main/recipe/meta.yaml